functions/security/idtoken.go

```diff
@@ -27,14 +26,21 @@ import (
 	"google.golang.org/api/idtoken"
 )
 
-// makeGetRequest makes a request to the provided targetURL with an authenticated client.
-func makeGetRequest(w io.Writer, targetURL string) error {
-	// functionURL := "https://TARGET_URL"
+// `makeGetRequest` makes a request to the provided `targetURL`
```

Comment:
The func docs should be the thing directly on-top-of the function. Region tags should either be before the docs or after the method.

It looks like this method does not follow our conventions for Go samples. Examples should be in the function body like: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#function-arguments-for-snippets

Reply:
Fixed.